lnprototest/clightning/clightning.py

```diff
@@ -221,13 +245,19 @@ def stop(self, print_logs: bool = False, also_bitcoind: bool = True) -> None:
         shutil.rmtree(os.path.join(self.lightning_dir, "regtest"))
 
     def restart(self) -> None:
+        # Preserve channel states across restarts
+        saved_states = self.channel_states
```

Comment:
```suggestion
        # Preserve channel states across restarts
        saved_states = self.channel_states
```
Mh this sounds wrong!

Reply:
I think you are correct, It wont do exactly what I thought it will do, will remove this.